ci-operator/step-registry/fips-check/fips-check-commands.sh

```diff
@@ -16,7 +23,7 @@ num_nodes="${#nodes[@]}"
 for (( i=0; i<$num_nodes; i++ )); do
   attempt=0
   while true; do
-      out=$(oc --request-timeout=60s -n default debug node/"${nodes[i]}" -- cat /proc/sys/crypto/fips_enabled || true)
+      out=$(oc --request-timeout=60s -n "$TEST_NS" debug node/"${nodes[i]}" -- cat /proc/sys/crypto/fips_enabled || true)
       if [[ ! -z "${out}" ]]; then
```

Comment:
Use `-n` instead of `! -z`.

Reply:
this PR is not about refactoring the script, but feel free to do that in a follow-up